main/params.go

```diff
@@ -152,6 +155,11 @@ func init() {
 
 	ferr := fs.Parse(os.Args[1:])
 
+	if *version { // If --version used, print version and exit
+		fmt.Println(node.Version.String())
```

Comment:
This currently prints `avalanche/0.5.1`. I propose
1. The name should match the name of the binary, e.g `ava/0.5.1`, `ava 0.5.1`. Goal is to reduce confusion for users unfamiliar with Avalanche terminology.
2. It would be nice to include some compatibility indications, e.g. `ava/0.5.1 (networkVersion=denali/3, dbVersion=0.5.0)`.

Reply:
1. I agree, I think we should change the version from `avalanche` -> `ava`. During the next incompatible update I think this should happen. However, changing the application name now would cause nodes on the network to have some confusion so I think it should stay this way for now.
2. I think that the currently displayed version is sufficient. The reason the dbVersion exists is so that we can do some quiet updates of the DB format without erasing anything, as erasing a DB could lose some information that the user wants to keep around... I'm happy to talk about this more, because I'm certainly not an expert here...

But I think this PR as is, is better than nothing and we have a release in -checks watch- 50 minutes. So, I'm going to merge this as is for now.

Reply:
I agree, made follow on tickets. I didn't intend that changing `-version` output would change on-the-wire network traffic.